httpbakery/form/form.go

```diff
@@ -123,9 +140,21 @@ func (v webPageVisitor) visitWebPage(u *url.URL) error {
 	req.Header.Set("Accept", "application/json")
 	var lm loginMethods
 	if err := v.client.Do(req, nil, &lm); err != nil {
```

Comment:
Interesting, although so far I haven't seen a need to chain to anything other that open a browser. It is probable it will happen.